posthog/models/feature_flag/flag_matching.py

```diff
@@ -390,13 +391,29 @@ def get_highest_priority_match_evaluation(
         return current_match, current_index
 
 
-def hash_key_overrides(team_id: int, person_id: int) -> Dict[str, str]:
+def hash_key_overrides(team_id: int, distinct_ids: List[str]) -> Dict[str, str]:
```

Comment:
NIT: Would be nice to rename to `get_hash_key_overrides`. It was confusing to read this because I couldn't tell if this function was supposed to retrieve or set until I reread that there's an explicit `set`

Reply:
makes sense!